contracts/governance/utils/VotesConfidential.sol

```diff
@@ -91,6 +111,14 @@ abstract contract VotesConfidential is Nonces, EIP712, IERC6372 {
         return _totalCheckpoints.upperLookupRecent(_validateTimepoint(timepoint));
     }
 
+    function getPastTotalSupplyWithAllowance(uint256 timepoint, bool persistent) public virtual returns (euint64) {
+        _validateVotesAccess(address(0));
```

Comment:
Are we allowing anyone to view total supply?

Reply:
https://github.com/OpenZeppelin/openzeppelin-confidential-contracts/blob/8bb34b874b84deb8cfcd83bcd355e6fca2c2a6f5/contracts/governance/utils/VotesConfidential.sol#L235-L240